subprojects/resources/src/main/java/org/gradle/internal/resource/DefaultTextFileResourceLoader.java

```diff
@@ -32,9 +32,6 @@ public TextResource loadFile(String description, @Nullable File sourceFile) {
         if (sourceFile == null) {
             return new StringTextResource(description, "");
         }
-        if (sourceFile.exists()) {
```

Comment:
Moved to https://github.com/gradle/gradle/pull/23822/files#diff-75fb303142f9c21ecab6b24529fc53751afdf68d01a7735d257c24347f2058afR90